packaging/helm-charts/helm3/strimzi-kafka-operator/templates/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -129,7 +129,7 @@ spec:
               value: {{ .Values.connectBuildTimeoutMs | quote }}
             {{- end }}
             {{- if .Values.extraEnvs }}
-            {{ toYaml .Values.extraEnvs | indent 12 }}
+{{ toYaml .Values.extraEnvs | indent 12 }}
```

Comment:
not an Helm expert but why the change is about changing the indentation of this line?

Reply:
It is due to the `indent 12`. I guess it starts counting the indentation space from the start